realtime-reporter/examples/realtime-reporter.yaml

```diff
@@ -0,0 +1,14 @@
+resources:
+  - apps/v1/deployments
+  - apps/v1/statefulsets
+  - batch/v1/jobs
+  - v1/pods
+  - v1/services
+  - networking.k8s.io/v1/ingresses
+  - v1/nodes
+  - v1/namespaces
+  - v1/persistentvolumes
+  - v1/persistentvolumeclaims
+
+namespaces:
+  - all
```

Comment:
What are the odds of some customer using `all` as an actual namespace?

Reply:
I'm not sure where this is explicit in https://github.com/kubernetes/client-go/blob/v0.29.1/dynamic/dynamicinformer/informer.go#L35, but providing `all` as a namespace works as intended, as it indicates that we want to watch all namespaces in the cluster. If you change this value to `default` or something else, it will be passed along when creating the dynamic informer and only report events occurring in the provided namespace(s)